### ISSUE TITLE
`vlanlist_to_config` fails on empy list (raises `IndexError`)

### ISSUE THREAD

**Issue by u1735067** [networktocode/netutils#194]
### Environment
* Python version: 3.10
* netutils version: 1.3.0

### Expected Behavior
`vlanlist_to_config` does not fails with a generic message (bug) / `vlanlist_to_config` allows an optional return value for empty list (fr)*

### Observed Behavior
`vlanlist_to_config` raises `IndexError` (like in https://github.com/ansible-collections/ansible.netcommon/issues/302)

### Steps to Reproduce
```py
>>> import netutils
>>> netutils.__version__
'1.3.0'

>>> import netutils.vlan
>>> netutils.vlan.vlanlist_to_config([])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/venv/lib/python3.10/site-packages/netutils/vlan.py", line 78, in vlanlist_to_config
    if vlan_list[0] < 1 or vlan_list[-1] > 4094:
IndexError: list index out of range
```

### Additional informations

You might be interested in https://github.com/aristanetworks/ansible-avd/blob/v3.8.0/ansible_collections/arista/avd/plugins/filter/list_compress.py and other filters in this project?

*: like this?
```py
DEFAULT_ARG = object()

def vlanlist_to_config(
    vlan_list: t.List[int],
    first_line_len: int = 48,
    other_line_len: int = 44,
    min_grouping_size: int = 3,
    empty_result: Any = DEFAULT_ARG
) -> t.List[str]:
[..]
if not vlan_list:
  if empty_result is DEFAULT_ARG:
    raise CustomExc('...')
  else:
    return empty_result
[..]
```

**Comment by itdependsnetworks** [comment]
Thanks for the report, should be able to fix easily.

**Comment by qduk** [comment]
Fixed by #219
